src/connectors/infinite-hits/__tests__/connectInfiniteHits-test.ts

```diff
@@ -842,220 +843,146 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/infinite-hi
     });
   });
 
-  describe('routing', () => {
-    describe('getWidgetState', () => {
-      it('should give back the object unmodified if the default value is selected', () => {
-        const renderFn = jest.fn();
-        const makeWidget = connectInfiniteHits(renderFn);
-        const widget = makeWidget({ showPrevious: true });
-
-        const helper = algoliasearchHelper({} as Client, '', {});
-        helper.search = jest.fn();
-
-        widget.init!(
-          createInitOptions({
-            state: helper.state,
-            helper,
-          })
-        );
-
-        const uiStateBefore = { page: 1 };
-        const uiStateAfter = widget.getWidgetState!(uiStateBefore, {
+  describe('getWidgetState', () => {
+    test('returns the `uiState` empty without `showPrevious` option', () => {
+      const render = jest.fn();
+      const makeWidget = connectInfiniteHits(render);
+      const helper = algoliasearchHelper(createSearchClient(), 'indexName');
+      const widget = makeWidget({});
+
+      const actual = widget.getWidgetState!(
+        {},
+        {
           searchParameters: helper.state,
           helper,
-        });
-
-        expect(uiStateAfter).toBe(uiStateBefore);
-      });
+        }
+      );
 
-      it('should add an entry equal to the refinement', () => {
-        const renderFn = jest.fn();
-        const makeWidget = connectInfiniteHits(renderFn);
-        const widget = makeWidget({ showPrevious: true });
+      expect(actual).toEqual({});
+    });
 
-        const helper = algoliasearchHelper({} as Client, '', {});
-        helper.search = jest.fn();
+    test('returns the `uiState` containing `page` with `showPrevious` option', () => {
+      const render = jest.fn();
+      const makeWidget = connectInfiniteHits(render);
+      const helper = algoliasearchHelper(createSearchClient(), 'indexName');
+      const widget = makeWidget({
+        showPrevious: true,
+      });
 
-        widget.init!(
-          createInitOptions({
-            state: helper.state,
-            helper,
-          })
-        );
+      const actual = widget.getWidgetState!(
+        {},
+        {
+          searchParameters: helper.state,
+          helper,
+        }
+      );
 
-        const { showMore } = renderFn.mock.calls[0][0];
+      expect(actual).toEqual({
+        page: 1,
+      });
+    });
 
-        showMore();
+    test('returns the `uiState` containing a page number incremented by one with `showPrevious` option and `page` search parameter', () => {
```

Comment:
```suggestion
    test('returns the `uiState` containing a page number incremented by one with `page` search parameter', () => {
```

We don't have to mention it on every tests, it's our default case.